components/omega/doc/devGuide/CondaEnv.md

```diff
@@ -7,45 +7,43 @@ code, you will need your own
 [conda](https://conda.io/projects/conda/en/latest/index.html) environment with
 the necessary packages.
 
-(omega-dev-install-mambaforge)=
+(omega-dev-install-miniforge3)=
 
-## Installing Mambaforge
+## Installing Miniforge3
 
 If you have not already done so, you will install
-[Mambaforge](https://github.com/conda-forge/miniforge#mambaforge), preferably
+[Miniforge3](https://github.com/conda-forge/miniforge#miniforge3), preferably
 somewhere in your home directory but possibly somewhere in your scratch space.
```

Comment:
````suggestion
[Miniforge3](https://github.com/conda-forge/miniforge#miniforge3):
```bash
curl -L -O "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-$(uname)-$(uname -m).sh"
bash Miniforge3-$(uname)-$(uname -m).sh
```
If you have the space to do so, we recommend installing Miniforge3 somewhere in our home 
directory.  Typically, this file system will be faster than on a scratch space spaces.  Also, a
conda environment will be rendered useless if its contents get purged.  However, conda
environments each require several GB of space, which may quickly use up your disk quota
so you will need to weigh these trade-offs for yourself.
````
@sbrus89, does that work for you?